eclipse-theia/theia
# repo: `yarn upgrade` following `1.18.0`

#### What it does


The commit perform a `yarn upgrade` to update dependencies pulled by our defined version ranges. The upgrade is important to identify any potential issues with our declared version ranges of dependencies and more accurately reflect the state for downstream applications that may upgrade themselves or build a fresh theia application.

#### Todo

- [ ] update `license:check` baseline

#### How to test


- confirm the application builds and starts
- confirm that there are no regressions

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
